package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dapr/dapr",
-  "version": "3.3.1",
+  "version": "3.4.0-rc.1",
```

Comment:
@DeepanshuA are we sticking with 3.3.1 to go with Dapr 1.14.0, or bumping to 3.4?  I see that the key bug fix you made for health probe went into 3.3.1.

Reply:
js-sdk ver 3.3.1 was released already. We would release 3.4.0 now.

Reply:
Once dapr 1.14 is released finally, will do so. As of now, dapr 1.14 rc versions are out, so creating js-sdk versions in sync with that.